example/src/react-barong/index.js

```diff
@@ -79,17 +87,24 @@ var BarongLoginForm = function (_a) {
 };
 
 var BarongRegisterForm = function (_a) {
-    var host = _a.host, redirection = _a.redirection;
+    var host = _a.host, redirection = _a.redirection, testMode = _a.testMode;
     var _b = useForm(), register = _b.register, handleSubmit = _b.handleSubmit, errors = _b.errors, watch = _b.watch;
     var onSubmit = useCallback(function (data) {
-        ApiUtil.register(host, data)
-            .then(function (response) {
-            response.status === 201 ? window.location.replace("" + redirection) : window.console.log(response);
-        })
-            .catch(function (error) {
-            window.console.log(error.response);
-        });
-    }, [host, redirection]);
+        if (testMode === true) {
+            window.location.replace(redirection);
+        }
+        else {
+            BarongApiUtil.register(host, data)
+                .then(function (response) {
+                response.status === 201
+                    ? window.location.replace("" + redirection)
```

Comment:
what is the purpose of `"" + redirection`
I guess using just `redirection` will be enough